### PR TITLE
feat: display canonical global nav on discourse

### PR DESCRIPTION
**Pull request by Jay-Topher**
## Done
- Fix blobal navbar display on discourse
- Driveby: Fix article page layout

### Desktop
image
image

### Mobile
image
image

### Article page (Before)
image

### Article page (After)
image

### CODE REVIEW

File: common/common.scss [hunk 1]
```diff
@@ -481,6 +296,221 @@ hr {
...
+.p-navigation--sliding .p-navigation__toggle--close:visited,
+.p-navigation--sliding .p-navigation__toggle--close:focus,
+.p-navigation--sliding .p-navigation__toggle--open,
+.p-navigation--sliding .p-navigation__toggle--open:hover,
+.p-navigation--sliding .p-navigation__toggle--open:visited,
+.p-navigation--sliding .p-navigation__toggle--open:focus,
+.p-navigation--sliding .p-navigation__dropdown-item,
+.p-navigation--sliding .p-navigation__dropdown-item:hover,
+.p-navigation--sliding .p-navigation__dropdown-item:visited,
+.p-navigation--sliding .p-navigation__dropdown-item:focus {
```
Comment (petermakowski) [inline common/common.scss:344]: We're using sass which means this could be simplified by nesting selectors, e.g.:

```scss
.p-navigation--sliding {
 .p-navigation__toggle--close,
 .p-navigation__toggle--open {
   &, &:hover, &:focus {
 // ...
 }
 }
 }
 ```

But most importantly, could you explain why is this necessary?

Comment (petermakowski) [reply common/common.scss:344]: As demonstrated in the vanilla framework docs, this style should be included in the framework already.

https://vanillaframework.io/docs/patterns/navigation

Google Chrome screenshot 001065@2x

Comment (Jay-Topher) [reply common/common.scss:344]: This is just to make sure that all links in every state has the black color, e.g: the `Menu` and `Close Menu` mobile navigation buttons.

Comment (Jay-Topher) [reply common/common.scss:344]: I agree though that thic can be nested, I'll work on this

Comment (Jay-Topher) [reply common/common.scss:344]: My thoughts also, it should have been included, but on inspecting and comparing with the global nav on maas.io, I realized that that wasn't the case.

Comment (Jay-Topher) [reply common/common.scss:344]: image
This is the menu button without the style

**Comment by Jay-Topher** [comment]
I'm having a look now, thanks

File: common/common.scss [hunk 2]
```diff
@@ -481,6 +296,204 @@ hr {
...
+  .p-navigation__toggle--open:hover,
+  .p-navigation__toggle--open:visited,
+  .p-navigation__toggle--open:focus,
+  .p-navigation__dropdown-item,
+  .p-navigation__dropdown-item:hover,
+  .p-navigation__dropdown-item:visited,
+  .p-navigation__dropdown-item:focus {
+    color: #000;
+  }
+}
```
Comment (petermakowski) [inline common/common.scss:329]: This still could be made more concise, `&, &:hover, &:visited, &:focus` pattern is used repeatedly.

**Review by petermakowski — APPROVED** [review]
